lib/holidays/date_calculator/lunar_date.rb

```diff
@@ -188,10 +341,18 @@ def to_s
       ].freeze
 
       MAX_YEAR_NUMBER = 150
+
+      # Given the region, CALENDAR_YEAR_INFO_MAP looks up the date 
+      # table and uses it in the calculation
       CALENDAR_YEAR_INFO_MAP = {
-        ko: KOREAN_LUNAR_YEAR_INFO
+        ko: KOREAN_LUNAR_YEAR_INFO,
```

Comment:
Could we switch to `:kr` so the symbols match with the definitions? What do you think?